community/docker/APKBUILD

```diff
@@ -2,15 +2,15 @@
 pkgname=docker
 pkgver=1.13.0
 _ver=${pkgver/_/-}
-pkgrel=0
+pkgrel=1
 _gitcommit=v$_ver
 pkgdesc="Pack, ship and run any application as a lightweight container"
 url="http://www.docker.io/"
 arch="all"
 license="ASL 2.0"
 depends="iptables git xz"
```

Comment:
I dont know. It seems to be introduced with commit 4248d1a88bc, which was the first commit.